serverless-offline-elasticmq/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-offline-elasticmq",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
If we make the option required, so users need to now add in the version, we will need to bump this as at least a minor